jetty-core/jetty-client/src/main/java/org/eclipse/jetty/client/transport/HttpReceiver.java

```diff
@@ -447,6 +453,12 @@ private void terminateResponse(HttpExchange exchange, Result result)
         }
     }
 
+    @Override
+    public InvocationType getInvocationType()
+    {
+        return Invocable.getInvocationType(contentSource);
```

Comment:
Is it worthwhile working this out once when the contentSource is set?
Probably not... but maybe???